core/src/main/java/org/web3j/protocol/core/methods/response/EthBlock.java

```diff
@@ -84,9 +84,68 @@ public static class Block {
         private String baseFeePerGas;
         private String withdrawalsRoot;
         private List<Withdrawal> withdrawals;
+        private String blobGasUsed;
+        private String excessBlobGas;
 
         public Block() {}
 
+        public Block(
+                String number,
+                String hash,
+                String parentHash,
+                String nonce,
+                String sha3Uncles,
+                String logsBloom,
+                String transactionsRoot,
+                String stateRoot,
+                String receiptsRoot,
+                String author,
+                String miner,
+                String mixHash,
+                String difficulty,
+                String totalDifficulty,
+                String extraData,
+                String size,
+                String gasLimit,
+                String gasUsed,
+                String timestamp,
+                List<TransactionResult> transactions,
+                List<String> uncles,
+                List<String> sealFields,
+                String baseFeePerGas,
+                String withdrawalsRoot,
+                List<Withdrawal> withdrawals,
+                String blobGasUsed,
+                String excessBlobGas) {
+            this.number = number;
+            this.hash = hash;
+            this.parentHash = parentHash;
+            this.nonce = nonce;
+            this.sha3Uncles = sha3Uncles;
+            this.logsBloom = logsBloom;
+            this.transactionsRoot = transactionsRoot;
+            this.stateRoot = stateRoot;
+            this.receiptsRoot = receiptsRoot;
+            this.author = author;
+            this.miner = miner;
+            this.mixHash = mixHash;
+            this.difficulty = difficulty;
+            this.totalDifficulty = totalDifficulty;
+            this.extraData = extraData;
+            this.size = size;
+            this.gasLimit = gasLimit;
+            this.gasUsed = gasUsed;
+            this.timestamp = timestamp;
+            this.transactions = transactions;
+            this.uncles = uncles;
+            this.sealFields = sealFields;
+            this.baseFeePerGas = baseFeePerGas;
+            this.withdrawalsRoot = withdrawalsRoot;
+            this.withdrawals = withdrawals;
+            this.blobGasUsed = blobGasUsed;
```

Comment:
For not supported blob networks, the value will be null for both blobGasUsed and excessBlobGas. I can initialise it will 0 if needed as - 
`For the first post-fork block, both parent.blob_gas_used and parent.excess_blob_gas are evaluated as 0.`